src/administration/troubleshooting.md

```diff
@@ -88,3 +88,60 @@ You will see a table like the following:
 | mastodon.coloradocrest.net | 6837196   | 6837196            | 0          | 1970-01-01 00:00:00+00        |
 
 This will show you exactly which instances are up to date or not.
+
+You can also use this website which will help you monitor this without admin rights, and also let others see it
+
+https://phiresky.github.io/lemmy-federation-state/site
+
+### You don't receive actions reliably
+
+Due to the lemmy queue, remove lemmy instances will be sending apub sync actions serially to you. If your server rate of processing them is slower than the rate the origin server is sending them, when visiting the [lemmy-federation-state](https://phiresky.github.io/lemmy-federation-state/site) for the remote server, you'll see your instance in the "lagging behing" section.
+
+Typically the speed at which you process an incoming action should be less than 100ms. If this is higher, this might signify problems with your database performance or your networking setup.
+
+Note that even apub action ingestion speed which seems sufficient for most other instances, might become insufficient if the origin server is receiving actions faster than you can process them. I.e. if the origin server receives 10 actions per second, but you can only process 8 actions per second, you'll inevitably start falling behind that one server only.
```

Comment:
> if the origin server is **sending** actions faster than you can process them

> the origin server **sends** 10 actions per second

right?

Reply:
No, I meant "receiving from their users", not "receiving via http". I'm trying to explain the dynamics of what would cause this holistically.